src/applications/find-forms/containers/SearchResults.jsx

```diff
@@ -224,6 +227,57 @@ export const SearchResults = ({
         {searchResults}
       </ul>
 
+      {/*  */}
+      <div
+        className="pdf-alert-modal"
+        style={{
+          height: '500px',
+        }}
+      >
+        <Modal
+          onClose={() => toggleModalState()}
+          secondaryButton={{
+            action: () => {
+              toggleModalState();
+            },
+            text: 'Close',
+          }}
+          title="Adobe Reader DC Required"
+          visible={isOpen}
+        >
+          <>
+            <p className="vads-u-display--block vads-u-margin-bottom--3">
+              <span>
+                All PDF forms do not function fully in a web browser or other
+                PDF viewer. Please download the form and use Adobe Acrobat
+                Reader DC to fill out. For specific instructions about working
+                with PDF’s
+              </span>{' '}
+              <a href="https://www.va.gov/resources/how-to-download-and-open-a-vagov-pdf-form">
+                please read out Resources and Support Article
+              </a>
+            </p>
+            <a
+              href="https://get.adobe.com/reader/"
```

Comment:
I think we probably want `rel="noopener noreferrer"` on links to external domains.

Reply:
Good catch brother!